CONTRIBUTING.rst

```diff
@@ -654,6 +654,7 @@ Here is the list of packages and their extras:
 Package                    Extras
 ========================== ===========================
 airbyte                    http
+alibabacloud               oss2
```

Comment:
What do you'll think if having `alibaba.cloud` similar to `google.cloud` and have similar sort of directory structure.

cc @eladkal

Reply:
@kaxil Yea. good idea :) I have renamed it to alibaba.cloud and resubmitted. Thx